fedimint-client/src/sm/executor.rs

```diff
@@ -170,7 +170,7 @@ impl Executor {
 
             dbtx.insert_entry(
                 &ActiveStateKey::from_state(state.clone()),
-                &ActiveState::new(),
```

Comment:
@elsirion  Should we rename `ActiveState` to something like `ActiveStateInfo`, `ActiveStateData`, `ActiveStateMeta`, etc.? The way things are RN it only contains `created_at`, the actual data is in the key, and we call it `meta: ActiveState` in a bunch of places, which was confusing me a lot.

Reply:
+1. This confused me as well when first looking at it. Should I change now or leave or separate PR?

Reply:
Not as bad of a change as I initially thought, changed these to `ActiveStateMeta` and `InactiveStateMeta` in a separate commit